rest_framework/templatetags/rest_framework.py

```diff
@@ -245,6 +245,29 @@ def items(value):
 
 
 @register.filter
+def schema_links(section, sec_key=None):
+    """
+    Recursively find every link in a schema, even nested.
+    """
+    NESTED_FORMAT = '%s > %s'
```

Comment:
What about for the JavaScript data binding in https://github.com/encode/django-rest-framework/blob/master/rest_framework/templates/rest_framework/docs/interact.html#L19 ?